### PR TITLE
Update 2.0 api doc including conv and kl loss

### PR DESCRIPTION
**Pull request by LielinJiang**
Update 2.0 api doc including conv and kl loss

### CODE REVIEW

**Comment by LielinJiang** [comment]
image
image
image
image
image
image
image
image
image
image

**Comment by LielinJiang** [comment]
image
image
image
image
image
image
image
image
image
image

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    - :math:`*` ：卷积操作
+    - :math:`b` ：偏置值，2-D Tensor，形状为 ``[M,1]``
+    - :math:`\sigma` ：激活函数
+    - :math:`Out` ：输出值，NCHW或NHWC格式的4-D Tensor， 和 ``X`` 的形状可能不同
+
+
+参数：
+    - **in_channels** (int) - 输入图像的通道数。
+    - **out_channels** (int) - 由卷积操作产生的输出的通道数。
+    - **kernel_size** (int|list|tuple) - 滤波器大小。如果它是一个列表或元组，则必须包含两个整数值：（kernel_size_height，kernel_size_width）。若为一个整数，kernel_size_height = kernel_size_width = kernel_size。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: 这样写是不是好一点：
卷积核大小。可以为单个整数或包含两个整数的元组或列表，分别表示卷积核的高和宽。如果为单个整数，表示卷积核的高和宽都等于该整数。

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    - **stride** (int|list|tuple，可选) - 步长大小。滤波器和输入进行卷积计算时滑动的步长。如果它是一个列表或元组，则必须包含两个整型数：（stride_height,stride_width）。若为一个整数，stride_height = stride_width = stride。默认值：1。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: 同上：
步长大小。可以为单个整数或包含两个整数的元组，分别表示卷积沿着高和宽的步长。如果为单个整数，表示沿着高和宽的步长都等于该整数。默认值为1。

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    - **padding** (int|list|tuple|str，可选) - 填充大小。如果它是一个字符串，可以是"VALID"或者"SAME"，表示填充算法，计算细节可参考上述 ``padding`` = "SAME"或  ``padding`` = "VALID" 时的计算公式。如果它是一个元组或列表，它可以有3种格式：(1)包含4个二元组：当 ``data_format`` 为"NCHW"时为 [[0,0], [0,0], [padding_height_top, padding_height_bottom], [padding_width_left, padding_width_right]]，当 ``data_format`` 为"NHWC"时为[[0,0], [padding_height_top, padding_height_bottom], [padding_width_left, padding_width_right], [0,0]]；(2)包含4个整数值：[padding_height_top, padding_height_bottom, padding_width_left, padding_width_right]；(3)包含2个整数值：[padding_height, padding_width]，此时padding_height_top = padding_height_bottom = padding_height， padding_width_left = padding_width_right = padding_width。若为一个整数，padding_height = padding_width = padding。默认值：0。
+    - **dilation** (int|list|tuple，可选) - 膨胀比例大小。空洞卷积时会使用该参数，滤波器对输入进行卷积时，感受野里每相邻两个特征点之间的空洞信息。如果膨胀比例为列表或元组，则必须包含两个整型数：（dilation_height,dilation_width）。若为一个整数，dilation_height = dilation_width = dilation。默认值：1。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: 这样是不是好一点：
空洞大小。可以为单个整数或包含两个整数的元组，分别表示卷积核中的元素沿着高和宽的空洞。如果为单个整数，表示高和宽的空洞都等于该整数。默认值：1。

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
+Conv2d
+-------------------------------
+
+.. py:class:: paddle.nn.Conv2d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, padding_mode='zeros', weight_attr=None, bias_attr=None, data_format="NCHW")
+
+
+
+**二维卷积层**
+
+该OP是二维卷积层（convolution2d layer），根据输入、滤波器、步长（stride）、填充（padding）、膨胀比例（dilations）一组参数计算输出特征层大小。输入和输出是NCHW或NHWC格式，其中N是批尺寸，C是通道数，H是特征高度，W是特征宽度。滤波器是MCHW格式，M是输出图像通道数，C是输入图像通道数，H是滤波器高度，W是滤波器宽度。如果组数(groups)大于1，C等于输入图像通道数除以组数的结果。详情请参考UFLDL's : `卷积 <http://ufldl.stanford.edu/tutorial/supervised/FeatureExtractionUsingConvolution/>`_ 。如果bias_attr不为False，卷积计算会添加偏置项。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: 滤波器->卷积核
膨胀比例->空洞大小

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+对每个输入X，有等式：
+
+.. math::
+
+    Out = \sigma \left ( W * X + b \right )
+
+其中：
+    - :math:`X` ：输入值，NCHW或NHWC格式的4-D Tensor
+    - :math:`W` ：滤波器值，MCHW格式的4-D Tensor
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: 滤波器->卷积核

File: doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+参数:
+    - **input** (Tensor) - KL散度损失算子的输入张量。维度为[N, \*]的多维Tensor，其中N是批大小，\*表示任何数量的附加维度，数据类型为float32或float64。
+    - **label** (Tensor) - KL散度损失算子的张量。与输入 ``input`` 的维度和数据类型一致的多维Tensor。
+    - **reduction** (str，可选) - 要应用于输出的reduction类型，可用类型为‘none’ | ‘batchmean’ | ‘mean’ | ‘sum’，‘none’表示无reduction，‘batchmean’ 表示输出的总和除以批大小，‘mean’ 表示所有输出的平均值，‘sum’表示输出的总和。
+    - **name** (str，可选) – 具体用法请参见 :ref:`api_guide_Name` ，一般无需设置。默认值：None。
+    
+返回：Tensor KL散度损失。
+
+返回类型：变量(Tensor)，数据类型与输入 ``input`` 一致。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst]: 不需要写返回类型，可以删掉

File: doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    import paddle.nn.functional as F
+
+    paddle.disable_static()
+
+    shape = (5, 20)
+    input = np.random.uniform(-10, 10, shape).astype('float32')
+    target = np.random.uniform(-10, 10, shape).astype('float32')
+
+    # 'batchmean' reduction, loss shape will be [N]
+    pred_loss = F.kl_div(paddle.to_variable(input),
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst]: to_variable -> to_tensor

File: doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+当 ``reduction``  为 ``none`` 时，输出损失与输入（x）形状相同，各点的损失单独计算，不会对结果做reduction 。
+
+当 ``reduction``  为 ``mean`` 时，输出损失为[1]的形状，输出为所有损失的平均值。
+
+当 ``reduction``  为 ``sum`` 时，输出损失为[1]的形状，输出为所有损失的总和。
+
+当 ``reduction``  为 ``batchmean`` 时，输出损失为[N]的形状，N为批大小，输出为所有损失的总和除以批量大小。
+
+参数:
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst:23]: 少了name参数

File: doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+    paddle.disable_static()
+
+    shape = (5, 20)
+    x = np.random.uniform(-10, 10, shape).astype('float32')
+    target = np.random.uniform(-10, 10, shape).astype('float32')
+
+    # 'batchmean' reduction, loss shape will be [N]
+    kldiv_criterion = nn.KLDivLoss(reduction='batchmean')
+    pred_loss = kldiv_criterion(paddle.to_variable(x),
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst]: 同上 to_tensor

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst (hunk 1 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: done, thanks

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst (hunk 2 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: done, thanks

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst (hunk 3 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: done, thanks~

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst (hunk 4 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: done, thanks

File: doc/fluid/api_cn/nn_cn/Conv2d_cn.rst (hunk 5 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/Conv2d_cn.rst]: done, thanks

File: doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst (hunk 6 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst]: done, thanks.

File: doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst (hunk 7 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/functional_cn/kl_div_cn.rst]: done, thanks

File: doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst (hunk 8 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst:23]: done, thanks

File: doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst (hunk 9 above)
Comment (LielinJiang) [reply doc/fluid/api_cn/nn_cn/loss_cn/KLDivLoss_cn.rst]: done, thanks

**Review by TCChenlong — APPROVED** [review]
LGTM
